### DISCUSSION TITLE
Feature request: enable multiple book selection in library window to perform removal of multiple publications

### DISCUSSION THREAD

**Discussion by danielweck** [edrlab/thorium-reader#1765] [Ideas]
(currently, deleting publications is cumbersome, one at a time)

**Comment by danielweck** [comment]
Related issue: https://github.com/edrlab/thorium-reader/issues/1131

**Comment by panaC** [comment]
with a checkbox on the cover ?

**Comment by danielweck** [comment]
> with a checkbox on the cover ?

Or maybe expose this additional "advanced" functionality only in the list view, in order to keep the grid view more basic?

**Comment by danielweck** [comment]
I'm moving this meta-issue to a "discussion" because the new bookshelf "data grid" (tabular column/rows visualisation) will evolve to include advanced features such as multiple selection. We need to flesh out details and subsequently file actionable issues associated with concrete development tasks.
